.vsts-dotnet-ci.yml

```diff
@@ -147,7 +147,7 @@ jobs:
 - job: CoreBootstrappedOnLinux
   displayName: "Linux Core"
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-latest'
```

Comment:
While we're here, I wonder if the mac os vmimage has an equivalent `macOS-10.14` to `macOS-latest`. Even if it does, I'd prefer CI PR's to be hyper-specific

Reply:
The answer to your question is here: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml